hpx/runtime/parcelset/parcelport_impl.hpp

```diff
@@ -753,6 +786,10 @@ namespace hpx { namespace parcelset
         /// The connection cache for sending connections
         util::connection_cache<connection, locality> connection_cache_;
 
+        typedef hpx::lcos::local::spinlock mutex_type;
+        mutex_type sender_threads_mtx_;
+        std::list<threads::thread_id_type> sender_threads_;
+
```

Comment:
How many threads do you expect to be active at the same time? Would a `std::set` be more appropriate to avoid the linear search (see L776)?